src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  font-size: 11px;
+  line-height: 13px;
```

Comment:
better to move default font styles to html

Reply:
done